src/neo-vm/ExecutionEngine.cs

```diff
@@ -866,6 +866,25 @@ private void ExecuteInstruction()
                         Push(Pop().GetInteger().Sqrt());
                         break;
                     }
+                case OpCode.MODMUL:
+                    {
+                        var modulus = Pop().GetInteger();
+                        var x2 = Pop().GetInteger();
+                        var x1 = Pop().GetInteger();
+                        Push(x1 * x2 % modulus);
```

Comment:
`MODMUL` is necessary?

Reply:
https://github.com/neo-project/neo-vm/issues/425#issuecomment-1109890871